README.md

```diff
@@ -1,122 +1,241 @@
+written to disk first. Then a new notification file is written to a temporary file,
+which is then renamed to avoid avoid race conditions resulting from overwriting this
+file as it's being served.
+
+Old snapshot and delta files are marked as 'deprecated' as soon they are no longer referenced
+by the current notification file. Files which have been deprecated for longer than N seconds
+are removed at the end of each synchronization (default 10 minutes).
+
+The default base directory for these files is `/var/lib/krill-sync/rrdp/`, but this
+can be overridden using the `--rrdp-dir` argument. The notification file will be
+called `notification.xml` and will be published directly under this base directory.
+Snapshot files are called `snapshot.xml`, delta files `delta.xml`. Snapshot and
+delta files are stored under two further directories based on the session ID, which
+is a uuid, and the serial.
+
+Example of a resulting structure:
 ```
-# krill-sync https://some.server/rrdp/notification.xml
+/var/lib/krill-sync/rrdp/notification.xml
+/var/lib/krill-sync/rrdp/de3b0c3c-85be-45f8-b89c-5d4a0fa66312/305278/snapshot.xml
+/var/lib/krill-sync/rrdp/de3b0c3c-85be-45f8-b89c-5d4a0fa66312/305278/delta.xml
+/var/lib/krill-sync/rrdp/de3b0c3c-85be-45f8-b89c-5d4a0fa66312/305277/delta.xml
+/var/lib/krill-sync/rrdp/de3b0c3c-85be-45f8-b89c-5d4a0fa66312/305276/delta.xml
 ```
 
-On first run this will download the complete set of RRDP snapshot and delta files from the remote server and store them locally, e.g. for serving by NGINX or some other HTTP server.
+Make sure that you map the base directory in your HTTPS front-end server, such
+as nginx, to this base directory in a way that the given public RRDP notification URI
+will map to the file on disk.
 
-On a subsequent run where no new data is available no changes will be made, with the possible exception of deletion of no longer referenced snapshot and delta files.
+## Rsync Data
 
-If new data is available the RRDP deltas will be used to update the local RRDP snapshot and delta files and then the Rsync files.
+Directory content for serving by an Rsync daemon is also created from the data downloaded
+via RRDP, no `rsync` binary is needed. If you do not need to serve the rsync content on
+a specific machine, then you can suppress this by using the `--disable-rsync` option.
 
-The Notification File is always written to disk last.
+This tool writes complete new rsync directories for each new RRDP session and serial that
+is retrieved. On unix systems symlinks are then used to link the `current` directory to
+the latest content. As it turns out the rsync daemon resolves symlinks whenever a client
+connects, so ongoing connections will keep being served the directory which was current
+at the time of connection.
 
-### Usage with docker
+The previous directory is marked as deprecated whenever a new current directory is
+created this way. At the end of each synchronization directories which have been
```

Comment:
s/synchronizaiton directories/synchronization run, directories/

Reply:
removed the docker section for now, until we have tested docker with 0.2